src/OrbitGl/ThreadTrack.cpp

```diff
@@ -250,9 +250,12 @@ Color ThreadTrack::GetTimerColor(const TimerInfo& timer_info, bool is_selected,
     color = orbit_gl::GetThreadColor(timer_info.thread_id());
   }
 
-  constexpr uint8_t kOddAlpha = 210;
+  constexpr uint8_t kOddRowColorMultiplier = 210;
   if ((timer_info.depth() & 0x1) == 0) {
-    color[3] = kOddAlpha;
+    // We are slightly alternating the colors for thread timers based on their depth.
+    for (int i = 0; i < 3; ++i) {
+      color[i] = static_cast<char>(static_cast<int>(color[i]) * kOddRowColorMultiplier / 255);
```

Comment:
Isn't this an integer division effectively resulting in 0? I think this would be more readable if you define `kOddRowColorMultiplier` as `float`, and you really only need the outermost cast in this case.

Reply:
> Isn't this an integer division effectively resulting in 0?

No. But I can make it `float` if you prefer :)

Reply:
Oh right, you're first multiplying, then dividing... my bad. Anyways, I think this is more readable. Thanks!